ngx-releng

```diff
@@ -237,7 +237,7 @@ if [ -d src ]; then
     tfiles=`find -L t -name '*.t'`
 
     ack -l '\r\n' $tfiles /dev/null
-    ack '^master_on' $tfiles /dev/null
+    #ack '^master_on' $tfiles /dev/null
```

Comment:
Why not remove it?

Reply:
https://github.com/openresty/lua-nginx-module/pull/1916

Reply:
I would like to add the auto check for the ci.
some cases should have master_on， checking for it is meaningless.

Reply:
I also think remove it simply could be better.

Reply:
has removed it